zanata-war/src/main/java/org/zanata/action/ProjectHome.java

```diff
@@ -1139,7 +1141,16 @@ private boolean checkViewObsolete() {
                 && identity.hasPermission("HProject", "view-obsolete");
     }
 
-    public class ProjectMaintainersAutocomplete extends MaintainerAutocomplete {
+    @ViewScoped
+    public static class ProjectMaintainersAutocomplete extends MaintainerAutocomplete {
```

Comment:
:+1: This is how we should declare all these helper instances. It will reduce the use of `ServiceLocator`.